implementation/test/TBTCSystemTest.js

```diff
@@ -69,55 +69,181 @@ describe("TBTCSystem", async function() {
     })
   })
 
-  describe("setSignerFeeDivisor", async () => {
-    it("sets the signer fee", async () => {
-      await tbtcSystem.setSignerFeeDivisor(new BN("201"))
+  describe("update Signer fee", async () => {
+    describe("beginSignerFeeDivisorUpdate", async () => {
+      const newFee = new BN("201")
+      it("executes and fires SignerFeeDivisorUpdateStarted event", async () => {
```

Comment:
Can we add a test case where we call begin update with different value once again after it has already been called?

Reply:
I think we should hold on this. 
from PR description:
```
Currently, the initial update functions are unlocked. This means that if a change is incoming and is halfway through the waiting period, the initial update function can be called again, the timer resets, an event is fired and the new change is now pending.
```

I'm not set on this functinality. Maybe we want to lock the function after an upgrade has been initiated, but there are drawbacks on both sides:

if we allow re-initiation at any time:
- function can be called all the time, emitting an event each time. 
if we don't allow re-initiation at any time:
- the risk of adding a time delay on sensitive System variables is amplified, since the time needed to change can be up to 2x timeDelay, or minimum of 2x if fixing a bad input.

Reply:
I'm not saying that we should block subsequent calls, I just wanted to have a test case that verifies we're able to do it.

Reply:
Alright, I'm good with testing it as is for now.